lib/pure/math.nim

```diff
@@ -1182,6 +1182,14 @@ func lcm*[T](x, y: T): T =
 
   x div gcd(x, y) * y
 
+func clamp*[T](val: T, rng: Slice[T]): T {.since: (1, 5), inline.}=
+  ## Like `system.clamp` but takes a slice so you can easily clamp within a range.
```

Comment:
```suggestion
func clamp*[T](val: T, rng: Slice[T]): T {.since: (1, 5), inline.} =
  ## Like `system.clamp`, but takes a slice, so you can easily clamp within a range.
```